### ISSUE TITLE
Question: Shouldn't ZF2 be a dependency?

### ISSUE THREAD

**Issue by localheinz** [ZF-Commons/ZfcBase#75]
I see this in `.travis.yml`:

``` yml
before_install:
 - cd test; git clone --depth=1 https://github.com/zendframework/zf2.git
```

meaning I can't run the tests locally unless I install ZF2. 

In my opinion, this should be a `dev` dependency, then.

**Comment by adamlundrigan** [comment]
@localheinz my only comment is that I would leave the existing autoloader setup in place.  
@Danielss89 what do you think is the likelihood that removing the autoloader would actually break someone's app?  Is there anyone at all left who's not using Composer? :P

**Comment by Danielss89** [comment]
Yes, i think there's alot of people who's not using composer, but just downloading.
